cmd/snap/cmd_download.go

```diff
@@ -29,13 +29,18 @@ import (
 
 	"github.com/snapcore/snapd/asserts"
 	"github.com/snapcore/snapd/asserts/sysdb"
+	"github.com/snapcore/snapd/cmd"
 	"github.com/snapcore/snapd/i18n"
 	"github.com/snapcore/snapd/image"
 	"github.com/snapcore/snapd/overlord/auth"
 	"github.com/snapcore/snapd/snap"
 	"github.com/snapcore/snapd/store"
 )
 
+func init() {
```

Comment:
I think I would slightly prefer putting it in main, it looks a bit arbitrary in here (but then, main is also arbitrary so no strong opinion).